src/common/AppNavbarMobile.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <div class="app-navbar-mobile">
+    <app-logo />
+    <button
+      class="app-navbar-mobile__burger-btn"
+      @click="isExtShown = !isExtShown"
+    >
+      <transition name="fade" mode="out-in">
+        <icon
+          class="app-navbar-mobile__burger-icon"
+          :key="String(isExtShown)"
+          :name="!isExtShown ? $icons.menuAlt1 : $icons.x"
+        />
+      </transition>
+    </button>
+    <transition name="fade" appear>
+      <div
+        v-if="isExtShown"
+        ref="backdropElement"
+        class="app-navbar-mobile__ext-wrp"
+        @click="onBackdropClick"
+      >
```

Comment:
- event listener on non-interactive element
- placing it as a separate element will remove your problem with checking whether click target was actually the backdrop element

Reply:
done